packages/react-composites/src/composites/MeetingComposite/state/MeetingAdapterState.ts

```diff
@@ -88,11 +23,9 @@ export interface MeetingAdapterClientState extends Pick<CallAdapterClientState,
   /** ID of the meeting participant using this Meeting Adapter. */
   userId: CommunicationIdentifier;
   /** Display name of the meeting participant using this Meeting Adapter. */
-  displayName: string;
-  /** Latest Errors that have occurred. */
-  latestErrors: MeetingErrors;
+  displayName: string | undefined;
   /** State of the current Meeting. */
-  meeting: MeetingState;
+  meeting: MeetingState | undefined;
```

Comment:
these is now `| undefined` to match Call/ChatAdapterClientStates that it is also `| undefined`